src/test/java/com/salesforce/dynamodbv2/testsupport/DefaultTestSetup.java

```diff
@@ -231,7 +231,7 @@ protected MtAmazonDynamoDbContextProvider getMtContext() {
                     .withKeySchema(new KeySchemaElement(RANGE_KEY_FIELD, KeyType.HASH))
                     .withProvisionedThroughput(new ProvisionedThroughput(1L, 1L))
                     .withProjection(new Projection().withProjectionType(ProjectionType.ALL)))
-                .withLocalSecondaryIndexes(null).build()
+                .withLocalSecondaryIndexes().build()
```

Comment:
Addressed "Confusing argument 'null', unclear if a varargs or non-varargs call is desired"